### PR TITLE
Fixes #31 VCR::Errors due to mismatch in API_BASE_URL in test setup when run locally

### LINKED ISSUE #31: VCR::Errors due to mismatch in API_BASE_URL in test setup
I just cloned this repository and noticed that the `spec/ioki/vcr` tests all failed with similar messages:

     VCR::Errors::UnhandledHTTPRequestError:

       ================================================================================
       An HTTP request has been made that VCR does not know how to handle:
         GET https://app.io.ki/api/platform/providers

The reason seems to be, that in the `spec/fixtures/vcr_cassettes` files the `https://demo.io.ki/api/` API_BASE_URL is used, but this is not configured by default in the test setup where `https://app.io.ki/api/` is used.

I was able to fix this by exporting the appropriate ENV variables as it is done in the `.github/ci_workflow.env` file:

```
export IOKI_PLATFORM_API_BASE_URL="https://demo.io.ki/api/"
export IOKI_PASSENGER_API_BASE_URL="https://demo.io.ki/api/" 
export IOKI_DRIVER_API_BASE_URL="https://demo.io.ki/api/"
```

This should be documented somewhere or maybe simply use the `https://app.io.ki/api/` API_BASE_URL in the VCR files as well and remove these ENV vars from the CI setup? Or set the ENV variable in the `spec_helper.rb` file?

### PR DESCRIPTION
**Pull request by eikes**

### CODE REVIEW

**Review by A5308Y — APPROVED** [review]

**Comment by A5308Y** [comment]
@ans-ioki How do you want to handle PR Reviews now that we are three people? Should we review and merge on our own or do you want to have a look at every PR?

**Comment by ans-ioki** [comment]
I think you all have enough experience to review each others code.
No code merge without tests. - Good coding standards. - Sane decisions.

I am not interested in reviewing everything just because it must be me. If another qualified coder is cross checking the PRs, they can get merged. Andreas H: You are finally "responsible" for this gem, if it is broken or stuck in development, has technical debt and so on, it is more of a longterm problem for you - so you should currently be the lead dev for this one. If you feel like you need input or a sparring review i am willing to help. But if i am ill or cant contribute even Robin can be a senior PR sparring partner.

So TL;DR - currently only seniors involved - any peer review is sufficient - lead dev / architect for ioki-ruby is Andreas H.

**Comment by A5308Y** [comment]
Thanks for clearing that up @ans-ioki and also thanks for the vote of confidence!
